vendor/github.com/containerd/fifo/handle_nolinux.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux
+// +build !linux,!windows
```

Comment:
Interesting; is it ignoring these because the file is _nolinux?

```
+ integration
# github.com/containerd/fifo
Error: ..\..\..\..\..\..\pkg\mod\github.com\containerd\fifo@v0.0.0-20210325135022-4614834762bf\handle_nolinux.go:34:11: undefined: syscall.Stat_t
Error: ..\..\..\..\..\..\pkg\mod\github.com\containerd\fifo@v0.0.0-20210325135022-4614834762bf\handle_nolinux.go:35:12: undefined: syscall.Stat
Error: ..\..\..\..\..\..\pkg\mod\github.com\containerd\fifo@v0.0.0-20210325135022-4614834762bf\handle_nolinux.go:49:11: undefined: syscall.Stat_t
Error: ..\..\..\..\..\..\pkg\mod\github.com\containerd\fifo@v0.0.0-20210325135022-4614834762bf\handle_nolinux.go:50:12: undefined: syscall.Stat
Error: ..\..\..\..\..\..\pkg\mod\github.com\containerd\fifo@v0.0.0-20210325135022-4614834762bf\mkfifo_nosolaris.go:24:9: undefined: syscall.Mkfifo
FAIL	github.com/containerd/containerd/integration/client [build failed]
mingw32-make: *** [Makefile:181: integration] Error 2
Error: Process completed with exit code 2.
```

Reply:
This was fixed in https://github.com/containerd/fifo/pull/39